tests/Serializer/BaseSerializationTest.php

```diff
@@ -1345,6 +1347,18 @@ public function testObjectWithNullableArrays()
         $this->assertEquals($this->getContent('nullable_arrays'), $this->serializer->serialize($object, $this->getFormat()));
     }
 
+    public function testGroupsTrim()
```

Comment:
I think that this tests in `tests/Serializer/BaseSerializationTest.php` are superflous